### ISSUE TITLE
[FEAT]Please add Daily Limits

### ISSUE THREAD

**Issue by zhixiangCai** [st3v3nmw/obsidian-spaced-repetition#505]
**Is your feature request related to a problem? Please describe.**
If I did not review my flashcards for somedays, the flashcards will accumulated a lot, wichi is frustrating.

**Describe the solution you'd like**
**Daily Limits**
New cards/day
Maximum reviews/day

![image](https://user-images.githubusercontent.com/62034759/193458146-1591b21b-ead8-4c07-94f7-5f4aef652efb.png)

**Additional context**

**Comment by wojtek1171** [comment]
Hi,
It's been a long time since there was a request for this feature. For my own use I prepared a quick solution which you can find here: 
https://github.com/wojtek1171/obsidian-spaced-repetition/commit/f5768cdb112679143471a8390dfd122a48777e09
It's definitely not an ultimate solution. The cards' limits reset after deck list reloads but I tried to make it as non-invasive as possible. 
@st3v3nmw  If you find that changes suitable let me know and I will open a PR. If not, let's leave it as unofficial hack :)

![Screenshot from 2023-08-19 21-26-03](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/19674576/87ea5774-e812-4cba-b557-44bb7c40ba9a)

**Comment by kaem-e** [comment]
Yes pleaseeee add this one i often review far more than 20 new notes per day and the current limit is just highly impractical

**Comment by notno** [comment]
I would like to see this too.

**Comment by mickvanhulst** [comment]
@wojtek1171 any progress on this, would love to see this feature supported.
